src/index.js

```diff
@@ -1,17 +1,16 @@
-// import React from 'react';
+// eslint-disable-next-line no-unused-vars
+import React from 'react';
+// eslint-disable-next-line import/extensions
 import ReactDOM from 'react-dom/client';
 import './index.css';
-// import App from './App';
-import reportWebVitals from './reportWebVitals';
+// eslint-disable-next-line no-unused-vars
+import App from './App.js';
```

Comment:
- [ ] Let's try to not disable Linter checks on our codebase as they are meant to help us write a clean and professional codebase. Disabling Linter checks defies its purpose. Kindly fix the errors 👍🏽

Reply:
Done